datasafe-business/src/test/java/de/adorsys/datasafe/business/impl/e2e/BaseStorageTest.java

```diff
@@ -89,4 +125,184 @@ private void validatePrivateStructAndEncryption(AbsoluteResourceLocation<Private
                 .filter(it -> pattern.test(it.location().toString()))
                 .collect(Collectors.toList());
     }
+
+    @Test
+    @SneakyThrows
+    @Ignore
+    public void writeToPrivateListPrivateInDifferentThreads() {
+        ThreadPoolExecutor executor = (ThreadPoolExecutor) Executors.newFixedThreadPool(4);
+        CountDownLatch holdingLatch = new CountDownLatch(1);
+        CountDownLatch finishHoldingLatch = new CountDownLatch(NUMBER_OF_TEST_USERS * NUMBER_OF_TEST_FILES);
+
+        FileInputStream input = new FileInputStream(new File(TEST_FILE));
+        String checksumOfOriginTestFile = checksum(input);
+        input.close();
+
+        String path = "folder2";
+
+        log.trace("*** Starting write threads ***");
+        for (int i = 0; i < NUMBER_OF_TEST_USERS; i++) {
+            long userRegisterTime = System.currentTimeMillis();
+            UserIDAuth user = registerUser("john_" + i, location);
+            loadReport.put("Register user " + user.getUserID().getValue(), System.currentTimeMillis() - userRegisterTime);
+            log.debug("Registered user: {}", user.getUserID().getValue());
+
+            createFileForUserParallelly(executor, holdingLatch, finishHoldingLatch, path, user, new AtomicInteger());
+
+            // open latch and start all threads
+            holdingLatch.countDown();
+        }
+
+        log.trace("*** Main thread waiting for all threads ***");
+        finishHoldingLatch.await();
+        executor.shutdown();
+        log.trace("*** All threads are finished work ***");
+
+        log.trace("*** Starting read info saved earlier *** ");
+        for (int i = 0; i < NUMBER_OF_TEST_USERS; i++) {
+            UserIDAuth user = createJohnTestUser(i);
+
+            List<AbsoluteResourceLocation<PrivateResource>> resourceList = services.privateService().list(forPrivate(user, "./")).collect(Collectors.toList());
+            log.debug("Read files for user: " + user.getUserID().getValue());
+
+            assertThat(resourceList.size()).isEqualTo(EXPECTED_NUMBER_OF_FILES_PER_USER);
+
+            resourceList.forEach(item -> {
+                String checksumOfDecryptedTestFile = calculateDecryptedContentChecksum(user, item);
+
+                log.info("Origin test file checksum hex: {}", checksumOfOriginTestFile);
```

Comment:
IMHO error message 
```
Expected :b
Actual   :a
```
that will be produced is enough, so we can get rid of log messages